SwiftificationTests/StringTests.swift

```diff
@@ -81,4 +81,10 @@ class StringTests: XCTestCase {
         }
     }
     
+    func testSafeSubscript() {
+        let str = "Hi"
+        XCTAssertEqual(str[str.startIndex.advancedBy(1)], str[safe: str.startIndex.advancedBy(1)]!)
```

Comment:
I almost commented on this too at first, ten thought, "oh, it's a test, so it's fine". :)